lib/ansible/modules/apt.py

```diff
@@ -63,44 +63,43 @@
     default: 'no'
   default_release:
     description:
-      - Corresponds to the C(-t) option for I(apt) and sets pin priorities
+      - Corresponds to the C(-t) option for C(apt) and sets pin priorities.
     aliases: [ default-release ]
     type: str
   install_recommends:
     description:
-      - Corresponds to the C(--no-install-recommends) option for I(apt). V(true) installs recommended packages.  V(false) does not install
+      - Corresponds to the C(--no-install-recommends) option for C(apt). V(true) installs recommended packages. V(false) does not install
         recommended packages. By default, Ansible will use the same defaults as the operating system. Suggested packages are never installed.
     aliases: [ install-recommends ]
     type: bool
   force:
     description:
-      - 'Corresponds to the C(--force-yes) to I(apt-get) and implies O(allow_unauthenticated=yes) and O(allow_downgrade=yes)'
-      - "This option will disable checking both the packages' signatures and the certificates of the
-        web servers they are downloaded from."
-      - 'This option *is not* the equivalent of passing the C(-f) flag to I(apt-get) on the command line'
+      - 'Corresponds to the C(--force-yes) to I(apt-get) and implies O(allow_unauthenticated=yes) and O(allow_downgrade=yes).'
+      - "This option will disable checking both the packages' signatures and the certificates of the web servers they are downloaded from."
+      - 'This option *is not* the equivalent of passing the C(-f) flag to C(apt-get) on the command line.'
       - '**This is a destructive operation with the potential to destroy your system, and it should almost never be used.**
          Please also see C(man apt-get) for more information.'
     type: bool
     default: 'no'
   clean:
     description:
       - Run the equivalent of C(apt-get clean) to clear out the local repository of retrieved package files. It removes everything but
-        the lock file from /var/cache/apt/archives/ and /var/cache/apt/archives/partial/.
+        the lock file from C(/var/cache/apt/archives/) and C(/var/cache/apt/archives/partial/).
       - Can be run as part of the package installation (clean runs before install) or as a separate step.
     type: bool
     default: 'no'
     version_added: "2.13"
   allow_unauthenticated:
     description:
       - Ignore if packages cannot be authenticated. This is useful for bootstrapping environments that manage their own apt-key setup.
-      - 'O(allow_unauthenticated) is only supported with O(state): V(install)/V(present)'
+      - 'O(allow_unauthenticated) is only supported with O(state=install) or O(state=present).'
```

Comment:
the previous version seemed more correct, as options and values are separately highlited

Reply:
Well, that seems to be a matter of personal preference. I am happy to revert that (here and other places I made the same change).